remote-controller-client/src/main/java/com/hazelcast/jet/tests/remote/RemoteControllerClient.java

```diff
@@ -53,6 +54,9 @@
     private static final int DEFAULT_PORT = 9701;
     private static final int VERIFICATION_DURATION_GAP = 15;
     private static final int SLEEP_BETWEEN_CLUSTER_RESTART_SECONDS = 30;
+    private static final int ASSERTION_RETRY_COUNT = 30;
```

Comment:
I suggest to increase it to 120 or 180 - we had some issues where some job get stuck for 1minute and then fail - this caused to fail all job on the cluster and not only the given job. We can also add logic to `assertWithRetry` which will log every unsuccessful assert iteration it it will be higher than some threshold for iteration (maybe higher than 30).